rclpy/rclpy/duration.py

```diff
@@ -16,10 +16,20 @@
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 
 
+CONVERSION_CONSTANT = 10 ** 9
```

Comment:
Perhaps change this to `NANOSECONDS_PER_SECOND` and add a comment noting that the type is `int` and not `float` like `1e9`

Reply:
Can do, of course, but I introduced this constant on the basis of https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/time.py#L20. Maybe better to keep these two sibling modules more alike?

Reply:
> Maybe better to keep these two sibling modules more alike?

This is generally a good idea, but the constant name here is not very descriptive - why propagate bad style? Having said that:

```suggestion
NANOSECONDS_PER_SECOND = 10 ** 9
```

Reply:
On second thought, dropped the constant altogether as the module does not need to mirror `rclpy.Time` that thoroughly.